data/ships.txt

```diff
@@ -203,7 +203,8 @@ ship "Bactrian"
 		"Ramscoop"
 		
 		"X4700 Ion Thruster"
-		"X5200 Ion Steering"
+		"X1700 Ion Thruster"
+		"X4200 Ion Steering"
```

Comment:
>And even if you can get away with stacking them, is steering not valued more than thrust on massive vessels like bactrians?

To that I say, are any of the ships really optimally outfitted in stock outfits?